VestingSchedule.sol

```diff
@@ -0,0 +1,227 @@
+pragma solidity ^0.5.3;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/UsingRegistry.sol";
+
+// Revision 1
+contract VestingSchedule is Ownable, UsingRegistry {
+    // solhint-disable not-rely-on-time
+
+    modifier onlyRevoker() {
+      require(msg.sender == _revoker, "sender must be the vesting revoker");
+      _;
+    }
+
+    modifier onlyBeneficiary() {
+      require(msg.sender == _revoker, "sender must be the vesting beneficiary");
+      _;
+    }
+
+    using SafeMath for uint256;
+
+    event VestingWithdrawn(address beneficiary, uint256 amount, uint256 timestamp);
+    event VestingRevoked(address revoker, address refundDestination, uint256 refundDestinationAmount, uint256 timestamp);
+
+    // amount that is to be vested
+    uint256 private _vestingAmount;
```

Comment:
I think all these fields should be public instead of private, so that they can be easily read

Reply:
done